easybuild/easyconfigs/p/pkgconfig/pkgconfig-1.2.2-foss-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'pkgconfig'
+version = '1.2.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://github.com/matze/pkgconfig'
+description = """pkgconfig is a Python module to interface with the pkg-config command line tool"""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'usempi': True}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['3685ba02a9b72654a764b728b559f327e1dbd7dc6ebc310a1bd429666ee202aa']
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('pkg-config', '0.29.2'),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
+}
+
+moduleclass = 'data'
```

Comment:
@boegel An observation - oughtn't this really to be `lib`? It's certainly not `data` in the sense that other `easyconfigs` in this PR are.

Reply:
`pkg-config` itself has `devel`, which certainly makes more sense than `data`; `lib` also does

Happy to change it, but let's make this a (trivial) separate PR to change all existing `pkgconfig` easyconfigs in one go?

Reply:
@boegel - yes, doesn't make sense to fix it all here. Just wanted to make sure it wasn't lost.